src/runner.js

```diff
@@ -12,6 +12,10 @@ export function run(stdout, stdin, argv) {
       'only the rules specified will be used to validate the schema. Example: FieldsHaveDescriptions,TypesHaveDescriptions'
     )
     .option(
+      '-c, --config-directory <path>',
+      'path to begin searching for config files.'
+    )
```

Comment:
Would love to hear your thoughts on this one.

I expected most people would run `graphql-schema-linter` from within their project's folder. Even if they aren't at the root (i.e. `/home/cjoudrey/my-project/tests/something`), `cosmiconfig` will move up directories until it finds a config or until it reaches home directory `~/`.